.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report 🐞
+about: Create a report to help us improve.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!-- This is not an exhaustive model but a help. No step is mandatory. -->
+
+**Description**
+Description of what the bug is about.
+
+**Expected behavior**
+What you expected to happen.
+
+**Current behavior**
+What happened.
+
+**Screenshots or Logs**
+If applicable, add screenshots or logs to help explain your problem.
+
+**Environment (please complete the following information):**
+- OS: [e.g. Debian GNU/Linux]
+- MeiliSearch version: [e.g. v.0.20.0]
+- firestore-meilisearch version: [e.g v0.1.1]
```

Comment:
It doesn't really matter, but since the version of this repo is `0.1.0` maybe you want to adapt the version here also :)